infrastructure/aws/production/ecs-task-definition-web.json

```diff
@@ -245,6 +245,10 @@
                     "name": "SSE_AUTHENTICATION_TOKEN",
                     "valueFrom": "arn:aws:secretsmanager:eu-west-2:084060095745:secret:ECS-API-LxUiIQ:SSE_AUTHENTICATION_TOKEN::"
                 },
+                {
```

Comment:
Have we added these secrete?

Reply:
Not yet, I don't have the credentials or authorization to do it. Could you?

Reply:
Have we added it to staging?

Reply:
I added instructions in the description on how to update the GitHub App once we have a secret defined and set.

Reply:
But we can't merge it without it, right?

Reply:
For another env var added @matthewelwell asked me to do first the PR declaring the env var in the infrastructure folder. And he set the value after that, however, I would say that he has the last word.

Reply:
Yeah, that makes sense, but we still can't merge it before the secrets are added because the deployment will fail

Reply:
@matthewelwell have set the env var secret.